style.css

```diff
@@ -382,6 +391,108 @@ button:disabled {
 }
 
 @media (width >= 768px) {
+  #header_container {
```

Comment:
Hello, @braincee 
how should I approach this part?
![image](https://github.com/Mehr-o-maah/Portfolio_2/assets/50089525/76bd5145-b0c1-4877-982f-d4e2e3bee45e)

I could make the card wider, but then it will not correspond to the margins in the Figma design. The same might happen with other elements, I would need to give up some margin or padding to fit the actual size of some elements in the figma design 🤔

Reply:
- [ ] Please you can approach it but add media queries with a minimum desktop breakpoint `min-width: 768px` in your CSS stylesheet.

Example: 

```
@media (min-width: 768px) {
   /* YOUR CODE GOES HERE*/
}
```

Reply:
@braincee but I already have the media query. The one you suggested means It is specifying that the code within the curly braces should apply only when the viewport width is equal to or exceeds 768 pixels. Which is the same as the one I wrote `@media (width >= 768px)` just with a different syntax